scripts/docker-compose.yml

```diff
@@ -119,12 +119,12 @@ services:
     depends_on:
       - insights-inventory-mq
     ports:
-      - 8081:8080
+      - 8001:8000
```

Comment:
For ROS and here, you mentioned port value  `8000`, will it create any problem?

Reply:
@kgaikwad I guess it slipped my mind that we have already modified the ports for concerned services in the file.
Could you try running the containers and ros api/processors once with this change, to see if it works for you well?

Reply:
Sure, I will verify it once. Is dev setup working fine on your local with mentioned ports change?

Reply:
Yeah its working in my local with the ports change :)